scripts/transfer_money.js

```diff
@@ -0,0 +1,67 @@
+// These are required to enable ES6 on tests
+// and it's dependencies.
+require('babel-register')({
+  ignore: /node_modules\/(?!zeppelin-solidity)/
+});
+require('babel-polyfill');
```

Comment:
Does it work if we remove this? It seems like the tests have been converted into CommonJS modules.

Reply:
Oh this file can be deleted entirely. Will do that now.